appveyor.yml

```diff
@@ -0,0 +1,16 @@
+version: '{build}'
+
+environment:
+    PATH:           'C:\Ruby200\bin;%PATH%'
+    VERSION_SUFFIX: -netstd
+
+install:
+    - bundle install

```

Comment:
I think should have `cmd:` in front of it. Perhaps that is the default, and that is why it works, but we've specified it explicitly elsewhere.